cleanupJavadoc.sh

```diff
@@ -1,2 +0,0 @@
-find release/doc -name "*.html" -exec sed -i "s/.*Generated by javadoc.*//" {} \;
```

Comment:
@pieroxy Agree with @mrDoctorWho on generated doc. Generated doc will become a nuisance to check in every time a change is made.